### ISSUE TITLE
Dataplane not reading certain tcp-request lines

### ISSUE THREAD

**Issue by KarlSorensenSky** [haproxytech/dataplaneapi#7]
I have a tcp-request section setup to do sni based routing but the dataplaneapi request to return the tcp-request rules seems to be missing one of the rules

My front end is configured as 
```
frontend test_frontend
  mode tcp
  bind *:8080

  acl clienthello req_ssl_hello_type 1
  tcp-request inspect-delay 2s
  tcp-request content capture req.ssl_sni len 30
  log-format "%ci:%cp [%t] %ft %b/%s %Tw/%Tc/%Tt %B %ts %ac/%fc/%bc/%sc/%rc %sq/%bq sni_hdr:%[capture.req.hdr(0)]"

  tcp-request content accept if clienthello
```

when i query the dataplane it is not returning the tcp-request content capture line
```
curl --user admin:password "http://localhost:5555/v1/services/haproxy/configuration/tcp_request_rules?parent_type=frontend&parent_name=test_frontend"
{"_version":121,"data":[{"id":0,"timeout":0,"type":"inspect-delay"},{"action":"accept","cond":"if","cond_test":"clienthello","id":2,"type":"content"}]}
```

interestingly the id count skips from 0 to 2 so i'm assuming it is picking up the line when it scans the config file but doesn't report it.

**Comment by mjuraga** [comment]
The issue here is we only support [accept, reject] actions for tcp-request content for now, we do not support capture. The id skips so when you rewrite anything the capture action stays there. Supporting all actions is already in the roadmap.

**Comment by KarlSorensenSky** [comment]
Thanks, is there any estimate as to when it will be available?

**Comment by mjuraga** [comment]
We are not yet ready to announce an ETA for it as we are also collecting community feedback which might influence our roadmap...

If you want to provide a PR yourself I'll be happy to guide you through the process.

**Comment by mjuraga** [comment]
This is available from: 84b3a0a2afcfc06a66b51092150ada92cfa0183d . 2.0 release is coming soon too.
